### PR TITLE
Steps for test-169 were added. Issue#181

### PR DESCRIPTION
**Pull request by vfedrunov**
A few tests with number '@TEST-169' exist
The first of them works 
image

Other tests in '@request-history-file-detail-view' don't have enough test data. But they ready to run after adding the test data

### CODE REVIEW

File: specs/features/request-history-file-detail-view.feature [hunk 1]
```diff
@@ -10,38 +10,42 @@ Background:
...
-        When I click on a available file record with id '<fileId>'
+        When I select a valid '<datetimeFrom>' and '<datetimeTo>'
+        And I click on a available file record with id '<fileId>'
         Then the file detail view opens
         And the file result details and the sanitisation issues content is displayed to show item '<issue>'
         Examples:
-            | fileId                               | issue    |
-            | 11aded86-a598-4c42-a8c8-094c01012ae0 | 97976586 |
+            | fileId                               | issue    | datetimeFrom        | datetimeTo          |
+            | 11aded86-a598-4c42-a8c8-094c01012ae0 | 97976586 | 18/01/2021 19:00 PM | 18/01/2021 23:59 PM |
```
Comment (NadHodeGW) [inline specs/features/request-history-file-detail-view.feature:19]: The first one can be a existing file but all others should use a new file, so process a new file and view that after filtering the log using the 1 hour selection

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
Please see comment in feature file

File: specs/features/request-history-file-detail-view.feature (hunk 1 above)
Comment (NadHodeGW) [reply specs/features/request-history-file-detail-view.feature:19]: @vfedrunov please use the latest master to use the updated file-detail-view.feature

**Comment by vfedrunov** [comment]
TEST-169, TEST-169_2 were passed

@NadHodeGW please take a look:
No 'Embedded file content present.' item and no 'First XREF table entry malformed and will be regenerated' item are displayed in the detailed results
File id: 10826ad3-2971-4d93-bab2-96e209f1a730, 38101dc7-8c11-4245-b7ad-0ea8c17f98fa
image

File: specs/features/request-history-file-detail-view.feature [hunk 2]
```diff
@@ -10,24 +10,25 @@ Feature: request-history-file-detail-view
...
+        And The transaction with '<type>' type and '<risk>' risk is available in the transaction log
         When I click on the transaction record to open the detail view
-        Then The issues content is displayed on the details view to show issue '<issue>'
+        Then The issues content section is displayed on the details view
+        And Expanding the content section shows the issue "<issue>"
         Examples:
-            | file                  | issue                                      |
-            | structuralIssues.xlsx | End of stream 'wsSortMap1.xml' not reached |
+            | file                  | issue                                      | type | risk              |
+            | structuralIssues.xlsx | End of stream 'wsSortMap1.xml' not reached | Xlsx | Allowed by Policy |
```
Comment (NadHodeGW) [inline specs/features/request-history-file-detail-view.feature:31]: we don't need the type and risk it adds complication for not much; you just need to submit a file get the fileId, search for that id in the log and open the record. please leave the 2nd step as it was:
 And The transaction is available in the transaction log

**Comment by NadHodeGW** [comment]
@vfedrunov please use file: 'EmbeddedFile.pdf' then

**Comment by vfedrunov** [comment]
> @vfedrunov please use file: 'EmbeddedFile.pdf' then

@NadHodeGW 'EmbeddedFile.pdf' is absent in '/src/data/input/' in master, could you share this file?
https://github.com/filetrust/icap-management-ui-tests/tree/master/src/data/input

File: specs/features/request-history-file-detail-view.feature (hunk 2 above)
Comment (vfedrunov) [reply specs/features/request-history-file-detail-view.feature:31]: The file id is absent in the icap response when the file was unmodified

```
ICAP server:icap-client-qa-main.uksouth.cloudapp.azure.com, ip:192.168.65.3, port:1344

No modification needed (Allow 204 response)

ICAP HEADERS:
	ICAP/1.0 204 Unmodified
	Server: C-ICAP/0.5.7
	Connection: keep-alive
	ISTag: CI0001-2.1.1

```

Comment (NadHodeGW) [reply specs/features/request-history-file-detail-view.feature:31]: yes, no file is returned if NTFS is on relay and no id returns for 204 responses

Comment (vfedrunov) [reply specs/features/request-history-file-detail-view.feature:31]: File is returned and contains the id. But no id is returned in the response and logs. 
I can try to use the id from the file if it is not a bug, that the file is returned in this case.

image

Comment (vfedrunov) [reply specs/features/request-history-file-detail-view.feature:31]: `ICAP server:icap-client-qa-main.uksouth.cloudapp.azure.com, ip:192.168.65.3, port:1344


ICAP HEADERS:
	ICAP/1.0 200 OK
	Server: C-ICAP/0.5.7
	Connection: keep-alive
	ISTag: CI0001-2.1.1
	Encapsulated: res-hdr=0, res-body=235

RESPMOD HEADERS:
	HTTP/1.0 403 Forbidden
	Server: C-ICAP
	Connection: close
	Content-Type: text/html
	X-Adaptation-File-Id: ca05f0ef-dbec-432d-854f-d55d866507af
	Via: ICAP/1.0 mvp-icap-service-6c7c5f94-jqkhz (C-ICAP/0.5.7 Glasswall Rebuild service )

`
The id is available in the log also today

Comment (NadHodeGW) [reply specs/features/request-history-file-detail-view.feature:31]: it all depends on the policy in use. you need to check the policy applied.

Comment (vfedrunov) [reply specs/features/request-history-file-detail-view.feature:31]: @NadHodeGW 
Sorry, I cannot see the policy in the preconditions in the feature file. As far as I remember I should not update feature files, should I?

**Comment by vfedrunov** [comment]
`FAIL  | 3 passed, 1 failed   // 2m`
The fail reason is 'No Content Management Policy is displayed!'
image

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**